doc/guide/README.md

```diff
@@ -41,28 +41,40 @@ later, by reconfiguring and running `./build.sh stdlib` in
 After unpacking a release or checking out the source code from Github, let
 `$GERBIL_HOME` be the top directory of Gerbil.
 
-Then:
+If you are using the default configuration, can build Gerbil simply with:
+```bash
+$GERBIL_HOME/src/build.sh
+```
+
+On the other hand, if you need to override some configuration defaults,
+you will have to use the following (with your own set of configuration options):
 ```bash
 $ cd $GERBIL_HOME/src
-$ ./configure
+$ ./configure --prefix=/path/to/which/to/install/gerbil --gambit=/path/to/installed/gambit \
+  --enable-libxml --enable-libyaml --enable-mysql --enable-lmdb --enable-leveldb
```

Comment:
let's not be so explicit with these options, people are gonna start enabling them by accident :)
How about just `--enable-feature ...`